### DISCUSSION TITLE
Managing users of the local version of Penpot

### DISCUSSION THREAD

**Discussion by VladimirLevadnij** [uxbox/uxbox#1624] [Q&A]
Hello! I could not find information on how to manage registered Penpot users. I'm talking about the local version of Penpot.

The documentation has information on how to create a user, including using the command https://help.penpot.app/technical-guide/getting-started/

But it's not clear to me if Penpot has some kind of administrator section in which you can perform various service activities, including managing users? For example, see a list of users, block users, reset a user's password, and so on?

**Comment by Nahor** [comment]
There is no admin interface, you can't manage users yet.

https://github.com/penpot/penpot/discussions/2508#discussioncomment-4016308
https://github.com/penpot/penpot/discussions/2680

**Reply by niwinz** [reply]
There is still not admin available, there is the project:  https://github.com/penpot/penpot-admin that uses Django's facilities for build quickly the admin, but the current code still has some issues, so it not included by default on the dockerfile

**Comment by ricardogpinho** [comment]
I would also wanted to know how can you delete a profile.
I tried the python script provided in which I had to run `docker exec -ti penpot_penpot-backend_1 python3 ./manage.py` but it only allows to `('create-profile', 'update-profile', 'derive-password')`
